webapp/src/components/expanded_view/component.tsx

```diff
@@ -1350,6 +1350,7 @@ const ReactionOverlay = styled.div`
     display: flex;
     flex-direction: column;
     gap: 12px;
+    pointer-events: none;
```

Comment:
I guess this is what you mentioned before. Will events pass through without altering the stack order?

Reply:
Yep, which is what we want -- we don't want to mess with the z-index of the reactions or notifications or error message boxes, those have to be on top, but we don't want the div they're on to intercept the pointer events.

Reply:
Cool, will need to merge this into my work as well, thanks.